src/Sylius/Behat/Service/Helper/JavaScriptTestHelper.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Service\Helper;
+
+use Behat\Mink\Exception\ElementNotFoundException;
+use FriendsOfBehat\PageObjectExtension\Page\UnexpectedPageException;
+
+final class JavaScriptTestHelper implements JavaScriptTestHelperInterface
+{
+    private $sleepTime;
+
+    public function __construct(int $sleepTime)
```

Comment:
`$microsecondsInterval`? `$usleepInterval`? It'd be cool to have some descriptive name for this argument.